video_creation/final_video.py

```diff
@@ -22,6 +22,12 @@
 W, H = 1080, 1920
 
 
+def change_background_resolution():
+    global W
+    global H
+    W, H = settings.config["settings"]["background"]["width"], settings.config["settings"]["background"]["height"]
+
+
```

Comment:
nah, just add this in the `make_final_video` function

```python
W: int = settings.config["settings"]["background"]["width"]
H: int = settings.config["settings"]["background"]["height"]

if not W or not H:
    W, H = 1080, 1920
```

Reply:
Yeah this functionality does make sense

Reply:
May I ask what does 
`if not W or not H:
    W, H = 1080, 1920`
do? I'm a bit miffed by that?

Reply:
if it's somehow empty use default width and height